src/payment/strategies/adyenv2/adyenv2-initialize-options.ts

```diff
@@ -16,10 +16,15 @@ export default interface AdyenV2PaymentInitializeOptions {
     containerId: string;
 
     /**
-     * The location to insert the Adyen 3DS V2 component.
+     * @deprecated The location to insert the Adyen 3DS V2 component.
```

Comment:
When you deprecate this field, it means you'll need to use `additionalActionContainerId` in the places where you're using `threeDS2ContainerId` right now. i.e.: 

```ts
const options = this._getAdyenV2PaymentInitializeOptions();

challengeComponent.mount(`#${options.additionalActionContainerId || options.threeDS2ContainerId}`);
```